### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 95)

### LINKED ISSUE #1462: [devtools-translate] Portuguese - What's New in DevTools (Chrome 95)
Post: https://developer.chrome.com/blog/new-in-devtools-95/

Translator: @alvarocamillont
Reviewer: @khaosdoctor

Stable rollout: Oct 12
Translation ready by: Oct 04

### PR DESCRIPTION
**Pull request by alvarocamillont**
Fixes #1462

Changes proposed in this pull request:

    Add portuguese translations for What's New in DevTools M95

### CODE REVIEW

**Comment by chybie** [comment]
Thanks Alvaro, @khaosdoctor, @andreban please help to review!

File: site/pt/blog/new-in-devtools-95/index.md [hunk 1]
```diff
@@ -29,53 +28,61 @@ tags:
...
-
+## Nova ferramenta de edição de comprimentos CSS {: #length }
 <!-- DevTools added an easier yet flexible way to update lengths in CSS! -->
-
+DevTools adicionou uma maneira mais fácil e flexível de atualizar comprimentos em CSS!
 <!-- In the **Styles** pane, look for any CSS property with length (e.g. `height`, `padding`). -->
-
+No painel **Styles**, procure qualquer propriedade CSS com comprimento (por exemplo, `height`,` padding`).
 <!-- Hover over the unit type, and notice the unit type is underlined. Click on it to select a unit type from the dropdown. -->
+Passe o mouse sobre o tipo de unidade e observe que o tipo de unidade está sublinhado. Clique nele para selecionar um tipo de unidade no menu suspenso.
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-95/index.md:37]: Question: is `menu suspenso` a well known translation for `dropdown menu`? This is the first time hearing this, but I may just not have heard this before.

**Review by andreban — APPROVED** [review]
Just one question, but otherwise LGTM! Great work!

File: site/pt/blog/new-in-devtools-95/index.md (hunk 1 above)
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-95/index.md:37]: I've seen both just `dropdown` and `menu suspenso` in translations

**Comment by andreban** [comment]
Yeah, it seems Excel uses "menu suspenso". LGTM then.

File: site/pt/blog/new-in-devtools-95/index.md [hunk 2]
```diff
@@ -29,53 +28,61 @@ tags:
...
 
 <!-- You can still edit the unit value as text — just click on the value and start editing. -->
+Você ainda pode editar o valor da unidade como texto - basta clicar no valor e começar a editar.
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/hBk2t2DCX7aI5yBX4J8h.mp4", autoplay="true", muted="true", loop="true", class="screenshot" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/20932ec7ffa980023548e6f9d010ba11d0a3eab7 #}
 
-Chromium issues: [1126178](https://crbug.com/1126178), [1172993](https://crbug.com/1172993)
+Issue relacionadas: [1126178](https://crbug.com/1126178), [1172993](https://crbug.com/1172993)
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-95/index.md:53]:
```suggestion
Issues relacionadas: [1126178](https://crbug.com/1126178), [1172993](https://crbug.com/1172993)
```

File: site/pt/blog/new-in-devtools-95/index.md [hunk 3]
```diff
@@ -88,75 +95,86 @@ link.search = `?access=${link.access}`;
...
-
+## Lighthouse 8.4 no painel Lighthouse {: #lighthouse }
 <!-- The **Lighthouse** panel is now running Lighthouse 8.4. Lighthouse will now detect if the [Largest Containful Paint (LCP)](https://web.dev/lcp) element was a lazy-loaded image and recommend removing the `loading` attribute from it. -->
+O painel **Lighthouse** agora executa o Lighthouse 8.4. O Lighthouse agora detectará se o elemento [Largest Containful Paint (LCP)](https://web.dev/lcp) era uma imagem de carregamento lazy e recomendará a remoção do atributo `loading` dela.
 
 <!-- Check out the [What’s new in Lighthouse 8.4](/blog/lighthouse-8-4/) for more details on the updates. -->
+Confira [O que há de novo no Lighthouse 8.4](/blog/lighthouse-8-4/) para obter mais detalhes sobre as atualizações. 
 
-{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="The lazy-loaded LCP audit in a Lighthouse report", width="800", height="502", class="screenshot" %}
+{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="A auditoria LCP com um elemento de carregamento lazy em um relatório Lighthouse", width="800", height="502", class="screenshot" %}
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-95/index.md:122]:
```suggestion
{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="A auditoria LCP carregada em lazy-loading em um relatório Lighthouse", width="800", height="502", class="screenshot" %}
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Minor changes

File: site/_includes/partials/devtools/pt/whats-new.md [hunk 4]
```diff
@@ -2,6 +2,15 @@
 
 Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no">What's New In DevTools</a> para uma lista completa dos recursos lançados. Abaixo estão alguns conteúdos que foram traduzidos para o português.
 
+### Chrome 95 {: #chrome95 }
+
```
[the remark this answers is not in the document]
Comment (alvarocamillont) [reply site/_includes/partials/devtools/pt/whats-new.md:6]: I credited the article, in this file it's just the abstract

**Comment by chybie** [comment]
Obrigado a todos!

**Pull request merged**